homeassistant/components/zwave_js/api.py

```diff
@@ -497,6 +506,80 @@ async def websocket_get_log_config(
     )
 
 
+@websocket_api.require_admin  # type: ignore
+@websocket_api.async_response
+@websocket_api.websocket_command(
+    {
+        vol.Required(TYPE): "zwave_js/opt_in_data_collection",
+        vol.Required(ENTRY_ID): str,
+    },
+)
+async def websocket_opt_in_data_collection(
+    hass: HomeAssistant, connection: ActiveConnection, msg: dict
+) -> None:
+    """Opt in to statistics collection."""
+    entry_id = msg[ENTRY_ID]
+    try:
+        update_data_collection_preference(hass, entry_id, True)
+    except NotFoundError as err:
+        connection.send_error(msg[ID], ERR_NOT_FOUND, err.args[0])
+        return
+    await async_enable_statistics(hass.data[DOMAIN][entry_id][DATA_CLIENT])
+    connection.send_result(
+        msg[ID],
+    )
+
+
+@websocket_api.require_admin  # type: ignore
+@websocket_api.async_response
+@websocket_api.websocket_command(
+    {
+        vol.Required(TYPE): "zwave_js/opt_out_data_collection",
+        vol.Required(ENTRY_ID): str,
+    },
+)
+async def websocket_opt_out_data_collection(
```

Comment:
I'd merge the opt in and opt out commands and add a boolean parameter to mark the choice.